### ISSUE TITLE
ICLR 2020

### ISSUE THREAD

**Issue by amyxie361** [amyxie361/Blog#25]
[Lite Transformer with Long-Short Range Attention](https://openreview.net/forum?id=ByeMPlHKPH)
MIT&SJTU
![image](https://user-images.githubusercontent.com/26669286/80898467-c9357a00-8cd1-11ea-84ff-8a0edc0d4a58.png)
![image](https://user-images.githubusercontent.com/26669286/80898471-cfc3f180-8cd1-11ea-9433-49be5674607b.png)
问题：减少transformer计算量/训练量
方法：
1. flatten FFN - 不用bottle neck
2. 减半attention量，加GRU-CNN-FC
效果：
1. attention确实不再diagnoal
2. 少计算量，好效果
![image](https://user-images.githubusercontent.com/26669286/80898513-48c34900-8cd2-11ea-8cb0-a9c564d23f17.png)

**Comment by amyxie361** [comment]
[Transformer-XH: Multi-Evidence Reasoning with eXtra Hop](Attentionhttps://openreview.net/forum?id=r1eIiCNYwS)
![image](https://user-images.githubusercontent.com/26669286/80898536-a192e180-8cd2-11ea-8278-0e186483fca9.png)
![image](https://user-images.githubusercontent.com/26669286/80898593-3a296180-8cd3-11ea-942d-4228d1778f71.png)
问题：multi-hopQA / add inter-doc relation into Transformer
方法：
1. feed inlink' doc representation into the first token of the next layer 
2. use retrival and wiki-link to form the graph
result:
SOTA on HotpotQA &FEVER
My Question:
what if each node has multiple inlink?
Is the path length at most the depth of the Transformer?
为什么要在不同transformer层之间传，不会迷惑掉transformer每层学习的功能吗

**Comment by amyxie361** [comment]
[Learning to Retrieve Reasoning Paths over Wikipedia Graph for Question Answering](https://openreview.net/pdf?id=SJgVHkrYDH)
![image](https://user-images.githubusercontent.com/26669286/80898776-84134700-8cd5-11ea-9e25-892a5f0814e4.png)
用HotpotQA的打法打OpenQA？
不知道和peng讲的那篇比创新点在哪

**Comment by amyxie361** [comment]
Dynamically Pruned Message Passing Networks for Large-scale Knowledge Graph Reasoning
https://openreview.net/pdf?id=rkeuAhVKvB

问题：knowledge graph reasoning
方法：用attention sampling subgraph
没咋听懂

**Comment by amyxie361** [comment]
Improving Neural Language Generation with Spectrum Control
![image](https://user-images.githubusercontent.com/26669286/80899117-2c2b0f00-8cda-11ea-8d6b-4d84bb5720d5.png)
![image](https://user-images.githubusercontent.com/26669286/80899139-685e6f80-8cda-11ea-8a92-bef4b65d0b34.png)
用可控制的SVD分解guide embedding matrix W的学习来控制生成？

**Comment by amyxie361** [comment]
Incorporating BERT into Neural Machine Translation
![image](https://user-images.githubusercontent.com/26669286/80899255-21717980-8cdc-11ea-8be9-7dbbbaf836f3.png)
借用BERT来翻译

**Comment by amyxie361** [comment]
StructBERT Alibaba
1. shuffle word order
2. NSP -> both next sentence and previous sentence

**Comment by amyxie361** [comment]
BERTscore

**Comment by amyxie361** [comment]
Strategies for Pre-training Graph Neural Networks
![image](https://user-images.githubusercontent.com/26669286/81766022-436cb800-94a3-11ea-8195-ac9114ec67f9.png)
1. unlabeled: node representation, attribute
2. labeled: graph representation
3. downstream tasks

**Comment by amyxie361** [comment]
![image](https://user-images.githubusercontent.com/26669286/81766787-d0fcd780-94a4-11ea-8382-c94124cf96fd.png)

**Comment by amyxie361** [comment]
The Curious Case of Neural Text Degeneration

top p sampling

**Comment by amyxie361** [comment]
Cross-lingual Alignment vs Joint Training: A Comparative Study and A Simple Unified Framework
align embedding
Q: mBERT对应句子的emcodding效果怎么样

**Comment by amyxie361** [comment]
[Data-dependent Gaussian Prior Objective for Language Generation](https://openreview.net/pdf?id=S1efxTVYDr)
解决MLE作为LM target时候的 negative sample loss 不恰当的问题
方法：加入word embedding空间里的gaussian prior作为loss 
效果：提高了BLEU/ppl，+diversity，+非常见词

**Comment by amyxie361** [comment]
NEURQURI: NEURAL QUESTION REQUIREMENT INSPECTOR FOR ANSWERABILITY PREDICTION
IN MACHINE READING COMPREHENSION

用query中关键span和context做varivication

TODO

**Comment by amyxie361** [comment]
Depth-Adaptive Transformer
https://openreview.net/forum?id=SJg7KhVKPH

TODO like early exit
